src/store/src/Bridge/Meilisearch/Store.php

```diff
@@ -28,17 +28,25 @@ final class Store implements ManagedStoreInterface, StoreInterface
 {
     /**
      * @param string $embedder        The name of the embedder where vectors are stored
-     * @param string $vectorFieldName The name of the field int the index that contains the vector
+     * @param string $vectorFieldName The name of the field in the index that contains the vector
+     * @param float  $semanticRatio   The ratio between semantic (vector) and full-text search (0.0 to 1.0)
+     *                                - 0.0 = 100% full-text search
+     *                                - 0.5 = balanced hybrid search
+     *                                - 1.0 = 100% semantic search (vector only)
      */
     public function __construct(
-        private readonly HttpClientInterface $httpClient,
-        private readonly string $endpointUrl,
-        #[\SensitiveParameter] private readonly string $apiKey,
-        private readonly string $indexName,
-        private readonly string $embedder = 'default',
-        private readonly string $vectorFieldName = '_vectors',
-        private readonly int $embeddingsDimension = 1536,
+        private HttpClientInterface $httpClient,
+        private string $endpointUrl,
+        #[\SensitiveParameter] private string $apiKey,
+        private string $indexName,
+        private string $embedder = 'default',
+        private string $vectorFieldName = '_vectors',
+        private int $embeddingsDimension = 1536,
+        private float $semanticRatio = 1.0,
```

Comment:
i guess that got lost with rebase

```suggestion
        private readonly HttpClientInterface $httpClient,
        private readonly string $endpointUrl,
        #[\SensitiveParameter] private readonly string $apiKey,
        private readonly string $indexName,
        private readonly string $embedder = 'default',
        private readonly string $vectorFieldName = '_vectors',
        private readonly int $embeddingsDimension = 1536,
        private readonly float $semanticRatio = 1.0,
```